src/components/power-sensor/client.test.ts

```diff
@@ -20,44 +20,24 @@ beforeEach(() => {
     .fn()
     .mockImplementation((_req, _md, cb) => {
       cb(null, {
-        getVolts: () => {
-          return {
-            Volts: 1,
-          };
-        },
-        getIsAc: () => {
-          return {
-            IsAc: true,
-          };
-        },
+        getVolts: () => 1.5,
+        getIsAc: () => true,
```

Comment:
e.g. if you agree with my suggestion above - same for the ones below
```suggestion
        getVolts: () => testVoltage,
        getIsAc: () => testIsAc,
```